.gitignore

```diff
@@ -23,4 +25,4 @@ yarn-debug.log*
 yarn-error.log*
 
 # Firebase Service account key
-streetcare-d0f33-firebase-adminsdk-idx6g-e46500ba2b.json 
\ No newline at end of file
+streetcare-d0f33-firebase-adminsdk-idx6g-e46500ba2b.json 
```

Comment:
What is this line?

Reply:
Hi Shardul, it's Firebase Service account key. It was previously present as well, just the position in the file has changed as I've added two more lines to the .gitignore file to ignore all the .idea files and node_modules files from any commit.

Reply:
Ok, great